addons/binding/org.openhab.binding.rfxcom/src/main/java/org/openhab/binding/rfxcom/internal/messages/RFXComMessage.java

```diff
@@ -46,6 +47,15 @@
     byte[] decodeMessage() throws RFXComException;
 
     /**
+     * Procedure for converting RFXCOM value to openHAB command.
+     *
+     * @param valueSelector
+     *
+     * @return Openhab command.
+     */
+    Command convertToCommand(RFXComValueSelector valueSelector) throws RFXComException;
```

Comment:
To me it feels like a work around to introduce seperate logic for the command, why can't we solve it without this method?